src/oic/oic/provider.py

```diff
@@ -2197,7 +2197,13 @@ def end_session_endpoint(
             try:
                 _ruri = self.cdb[client_id]["post_logout_redirect_uris"]
             except KeyError:
-                redirect_uri = self.post_logout_page
+                if self.post_logout_page is None:
+                    logger.info("No post logout page configured for %s", client_id)
+                    return error_response(
+                        "server_error", "Have no post logout page configured"
```

Comment:
I am not sure I am reading this flow correctly. Does this actually log the user out or not?

Reply:
It doesn't log the user out.
The user is never sent to the verify logout page. The actual log out doesn't happen until the user has said yes at that page.
So, not being sent to the verify logout page means no logout.

Reply:
OK, just wanted to be sure.